<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>segundaWindow</class>
 <widget class="QMainWindow" name="segundaWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnMenu2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>MENÚ</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>UTEC.png</normaloff>UTEC.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnIzquierda2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>81</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>flecha-hacia-la-izquierda-antes-de-icono-4330-128.png</normaloff>flecha-hacia-la-izquierda-antes-de-icono-4330-128.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEconomico2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>521</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>Económico </string>
    </property>
   </widget>
   <widget class="QPushButton" name="bntEstudiantil2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>521</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Estudiantil</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEjecutivo2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>521</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>Ejecutivo</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>180</y>
      <width>101</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>6.00</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>270</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>8.00</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>360</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>9.50</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>806</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMENUTEC">
    <property name="title">
     <string>MENUTEC</string>
    </property>
   </widget>
   <addaction name="menuMENUTEC"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
